js/webform_copy_value.js

```diff
@@ -17,18 +21,24 @@ jQuery(document).ready(function ($) {
   $("input[type=checkbox]").change(function() {
     copyValueToTargetCheckbox(this, sourceTargetMap);
   });
-  
-  $("input[type=radio]").change(function() {
+
+  $("input[type=radio]").click(function() {
```

Comment:
thanks for catching this @coldrunKacper , I removed change() as part of an experiment and forgot to put it back
Making the change now.